mlir/include/mlir/Dialect/SparseTensor/IR/SparseTensorAttrDefs.td

```diff
@@ -240,8 +240,9 @@ def SparseTensorEncodingAttr : SparseTensor_Attr<"SparseTensorEncoding",
     // CSR slice (offset = 0, size = 4, stride = 1 on the first dimension;
     // offset = 0, size = 8, and a dynamic stride on the second dimension).
     #CSR_SLICE = #sparse_tensor.encoding<{
-      lvlTypes = [ "dense", "compressed" ],
-      dimSlices = [ (0, 4, 1), (0, 8, ?) ]
+      map = (d0 : #sparse_tensor<slice(0, 4, 1)>,
```

Comment:
so the elaborate syntax would be

 (i = ib * 2 + ii  : #sparse_tensor<slice(0, 4, 1)>, ....

if that ever makes sense?

Reply:
Yeah, it's a bit hard to comprehend. But for now, let's keep it this way and we could optimize it later once the migrate is done.